nanshe_workflow/data.py

```diff
@@ -94,23 +90,21 @@ def _int_dask_rm_tree(dname):
 def dask_io_remove(name, executor=None):
     name = os.path.abspath(name)
 
-    rm_task = None
-    if not os.path.exists(name):
-        rm_task = dask_rm_nothing()
-    elif os.path.isfile(name):
-        rm_task = dask_rm_file(name)
-        rm_task = dask.delayed(io_remove)(rm_task)
-    elif os.path.isdir(name):
-        rm_task = dask_rm_tree(name)
-        rm_task = dask.delayed(io_remove)(rm_task)
-    else:
-        raise ValueError("Unable to remove path, '%s'." % name)
+    tmp_dir = "tmp_nanshe_workflow_{0}_".format(
+        os.path.splitext(os.path.basename(name))[0]
+    )
+    tmp_dir = tempfile.mkdtemp(prefix=tmp_dir)
+
+    if os.path.exists(name):
+        os.rename(name, os.path.join(tmp_dir, name))
```

Comment:
We may have to change this to `shutil.move` depending on how NFS reacts to this. FWICT it should be atomic (even on NFS). So this is the preferable operation to ensure that the contents are not in the way and can be found in the temporary directory afterwards.